test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -30,6 +31,8 @@ spec setup =
         result.to_vector.at 0
     do_round n dp=0 use_bankers=False = do_op n (_.round dp use_bankers)
```

Comment:
Yup. `do_op` creates a table with row `"x"` and single value `n` and then applies the operation to column `"x"` and returns the result.